WIP - Soporte para 1024

RecoX opened the pull request:
Soporte para 1024

Se mergea?

joaquinodz committed:
Soporte para mapas 680 y 1024

joaquinodz committed:
Update frmMain.frm

joaquinodz committed:
Mejoras y Fixes de @ReyarB

joaquinodz committed:
Fix minimap viewport.

joaquinodz committed:
Merge branch 'master' into soporte-para-1024

joaquinodz committed:
Fix build

Wyr0X committed:
Fix clase no registrada

RecoX commented on Codigo/clsDX8Engine.bas:
```diff
@@ -848,36 +848,38 @@ Sub RenderScreen(ByVal tilex As Integer, _
     Else
-        screenminX = 1
-        ScreenX = 1
+        screenminX = screenminX - 1 ' no se porque estaba asi pero lo dejo por las dudas
```
Esto me hace ruido, me parece que hay que revertir.

ReyarB replied:
=1 frena el traslado del mapa hacia abajo

ReyarB replied:
esta confirmado se pueden sacar los mentados

RecoX replied:
Lo que me hace ruido es que el `if` y el `else` hacen lo mismo haciendolo irrelevantes.

ReyarB replied:
Ahora recuerdo cuando modifique screenminY lo vi, por eso lo comente pero no lo borre porque no entendí el porque dos veces y lo deje para después y me olvide.

RecoX commented on Codigo/frmMain.frm:
```diff
@@ -1,6 +1,6 @@
 VERSION 5.00
-Object = "{F9043C88-F6F2-101A-A3C9-08002B2F49FB}#1.2#0"; "ijl11.dll"
-Object = "{3B7C8863-D78F-101B-B9B5-04021C009402}#1.2#0"; "RICHTX32.ocx"
+Object = "{F9043C88-F6F2-101A-A3C9-08002B2F49FB}#1.2#0"; "COMDLG32.OCX"
```
Object = "{F9043C88-F6F2-101A-A3C9-08002B2F49FB}#1.2#0"; "ijl11.dll"  

es eso necesario?

Wyr0X replied:
El nombre estaba mal, nunca supe por qué. El nombre debía ser COMDLG32.OCX. Supongo que algún error con VB. Igual no causaba problemas porque el id del objeto era el mismo, sólo era un error de nombre

RecoX commented on Codigo/frmMain.frm:
```diff
@@ -3994,52 +4032,86 @@ End Sub
+            TxTDerecha = 78
+            TxTIzquierda = 78
+            TxTAbajo = 87
+        Else
```
Indentar correctamente.

RecoX commented on Codigo/modEdicion.bas:
```diff
@@ -869,14 +871,37 @@ Sub DobleClick(tX As Integer, tY As Integer)
 
 ' Translados
+
+        If MapInfo.Changed = 1 Then
```
Tabulacion

ReyarB replied:
? solo actualiza que el mapa esta modificado?

RecoX replied:
Si, pero esta mal identado o tabulado

RecoX commented on Codigo/modEdicion.bas:
```diff
@@ -206,6 +206,8 @@ Public Sub Bloquear_Bordes()
 
     End If
+    
+Call Resolucion
```
Tabulacion

RecoX commented on Codigo/modEdicion.bas:
```diff
@@ -869,14 +871,37 @@ Sub DobleClick(tX As Integer, tY As Integer)
             frmMain.Dialog.FileName = PATH_Save & NameMap_Save & tTrans.Map & ".map"
-            Call modMapIO.MapaV2_Cargar(frmMain.Dialog.FileName)
+            
+        Select Case frmMain.Dialog.FilterIndex
```
Arreglar Tabulacion

ReyarB committed:
actualización de algunos indices

hay mas texturas para agregar esta es la primera parte.

ReyarB committed:
Modifico el pegado de bordes de mapas 

Este WE ahora sirve para editar mapas 1024 y 800 indicando que tipo de mapa estamos trabajando.
Modifico el pegado de mapas borrando mapa y pegando, antes sumaba al mapa ahora borra y pega.

ReyarB committed:
Elimino varios Index

ReyarB committed:
Mando de nuevo

Encontre errores mando nuevamente

ReyarB committed:
Indices

nuevos indices

ReyarB committed:
elimino esos If demas

ReyarB committed:
Sumo Indices

agrego mas indices con Grh ya indexados e indexo unas texturas mas a graficos.ini

ReyarB committed:
modifico numero de versión a 1.5.5

ReyarB committed:
Icono de AO libre

Versión 1.5.5
